docs/source/tutorials/object_detection.ipynb

```diff
@@ -498,6 +509,115 @@
     "visualize(image_path)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "46d6282a-4601-4cc3-b8a8-187ea6d5f8bc",
+   "metadata": {},
+   "source": [
+    "## Exploratory data analysys\n",
+    "\n",
+    "This section focuses on techniques to uncover annotation irregularities through exploratory data analysis. Specifically, it aims to highlight anomalies in object sizes, detect images with unusual object counts, and examine the distribution of class labels."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3335b8a3-d0b4-415a-a97d-c203088a124e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_imgs_to_show = 3\n",
+    "lab_object_counts,pred_object_counts = object_counts_per_image(labels,predictions)\n",
+    "for image_to_visualize in np.argsort(lab_object_counts)[::-1][0:num_imgs_to_show]:\n",
+    "    image_path = IMAGE_PATH + labels[image_to_visualize]['seg_map']\n",
+    "    print(image_path, '| idx', image_to_visualize)\n",
+    "    visualize(image_path, label=labels[image_to_visualize], class_names=class_names)"
+   ]
+  },
+  {
```

Comment:
add text cell before this code cell, which says:

"Next let's consider the distribution of class labels in the overall annotations, comparing the distribution in the given annotations vs. in the model predictions. This can sometimes reveal that something's off in our dataset or model."